engine/release-notes/24.0.md

```diff
@@ -19,6 +19,30 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.2
+
+{% include release-date.html date="2023-05-26" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.2 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.2)
+- [moby/moby, 24.0.2 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.2)
+
+### Bug fixes and enhancements
+
+* Fix a panic during build when referencing locally tagged images. [moby/buildkit#3899](https://github.com/moby/buildkit/pull/3899), [moby/moby#45582](https://github.com/moby/moby/pull/45582)
+* Fix builds potentially failing with `exit code: 4294967295` when performing many concurrent build stages. [moby/moby#45620](https://github.com/moby/moby/pull/45620)
+* Fix DNS resolution on Windows ignoring `etc/hosts` (`%WINDIR%\System32\Drivers\etc\hosts`), including resolution of `localhost`. [moby/moby#45562](https://github.com/moby/moby/pull/45562)
+* Apply a workaround for a containerd bug that causes concurrent `docker exec` commands to take significantly longer than expected. [moby/moby#45625](https://github.com/moby/moby/pull/45625)
+* containerd image store: Fix an issue where the image `Created` field would contain an incorrect value. [moby/moby#45623](https://github.com/moby/moby/pull/45623)
+* containerd image store: Adjust the output of image pull progress to match the output without the containerd image store enabled. [moby/moby#45602](https://github.com/moby/moby/pull/45602)
+* containerd image store: Prevent switching the containerd image store feature without restarting the daemon. [moby/moby#45616](https://github.com/moby/moby/pull/45616)
```

Comment:
```suggestion
* containerd image store: Fix an issue where the image `Created` field would contain an incorrect value. [moby/moby#45623](https://github.com/moby/moby/pull/45623)
* containerd image store: Adjust the output of image pull progress so that the output has the same format regardless of whether the containerd image store is enabled. [moby/moby#45602](https://github.com/moby/moby/pull/45602)
* containerd image store: Make the daemon restart when switching between the containerd image store and the default image store. [moby/moby#45616](https://github.com/moby/moby/pull/45616)
```

Reply:
Looking at the last bullet: "make the daemon restart" is probably not correct. The fix is that changing the option does not take effect until restarting the daemon. Not sure what the best wording is for that though; perhaps you have suggestions.

Reply:
aha, I misunderstood then. How about; "Switching between the default and containerd image store now requires a daemon restart"?

Reply:
Ah, yes, that works

Reply:
Updated 👍